test/service/test_app_logging.py

```diff
@@ -16,67 +20,236 @@
 from azul.logging import (
     configure_test_logging,
 )
+from azul.strings import (
+    trunc_ellipses,
+)
 from indexer import (
     DCP1CannedBundleTestCase,
 )
+from service import (
+    WebServiceTestCase,
+)
 
 
 # noinspection PyPep8Naming
 def setUpModule():
     configure_test_logging()
 
 
-class TestServiceAppLogging(DCP1CannedBundleTestCase, LocalAppTestCase):
+class TestServiceAppLogging(DCP1CannedBundleTestCase, WebServiceTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls._setup_indices()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._teardown_indices()
+        super().tearDownClass()
 
     @classmethod
     def lambda_name(cls) -> str:
         return 'service'
 
+    @property
+    def _head_response(self) -> str:
```

Comment:
This can be a normal class-level attribute. The only reason to use a property would be if `self` or some global, mutable state (uncommon) were to be accessed in the definition.